### PR TITLE
Removing not needed and incorrect ReferenceCopyLocalPaths references

### PR DESCRIPTION
**Pull request by moozzyk**
Fixes: #142

### CODE REVIEW

File: NuGet/Microsoft.AspNet.SignalR.Client.Cpp.WinDesktop.targets.template [hunk 1]
```diff
@@ -47,31 +47,27 @@
...
     </ItemGroup>
     <ItemGroup Label="x64 and #Toolset# and Release" Condition="'$(Platform.ToLower())' == 'x64' And '$(PlatformToolset.ToLower())' == '#Toolset#' And $(Configuration.ToLower().IndexOf('debug')) == -1">
       <CopyToOutput Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Release\signalrclient.dll" />
       <CopyToOutput Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Release\signalrclient.pdb" />
       <None Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Release\signalrclient.dll">
         <DeploymentContent>true</DeploymentContent>
       </None>
-      <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Debug\signalrclient.dll" />
     </ItemGroup>
     <ItemGroup Label="Win32 and #Toolset# and Debug" Condition="'$(Platform.ToLower())' == 'Win32' And '$(PlatformToolset.ToLower())' == '#Toolset#' And $(Configuration.ToLower().IndexOf('debug')) &gt; -1">
```
Comment (BrennanConroy) [inline NuGet/Microsoft.AspNet.SignalR.Client.Cpp.WinDesktop.targets.template]: Just curious, how does `Platform.ToLower()` ever equal `Win32`?

Comment (moozzyk) [reply NuGet/Microsoft.AspNet.SignalR.Client.Cpp.WinDesktop.targets.template]: wow.. good catch. Looks like the other bug concealed this one. I will try to see what happens and will update the PR...

Comment (moozzyk) [reply NuGet/Microsoft.AspNet.SignalR.Client.Cpp.WinDesktop.targets.template]: `.ToLower` is not needed - comparison is case insensitive.

**Comment by moozzyk** [comment]
:up::date: 
found one more issue - in release build both the debug and release versions of signalrclient.lib would be included.

**Comment by BrennanConroy** [comment]
:shipit:

**Pull request merged**